nolearn/lasagne/base.py

```diff
@@ -631,3 +643,29 @@ def _get_param_names(self):
         # This allows us to have **kwargs in __init__ (woot!):
         param_names = super(NeuralNet, self)._get_param_names()
         return param_names + self._kwarg_keys
+
+    @property
+    def custom_score(self):
+        if self.custom_scores and len(self.custom_scores) == 1:
```

Comment:
Appreciate the thoroughness, but I don't think we need to support backward compatibility for setting attributes on the `NeuralNet` object directly.  As far as I'm concerned, setting `net.custom_score` was never part of the API.